client/lib/gsuite/new-users.ts

```diff
@@ -132,8 +138,45 @@ const doesUserHaveError = ( user: GSuiteNewUser ): boolean => {
 const userIsReady = ( user: GSuiteNewUser ): boolean =>
 	isUserComplete( user ) && ! doesUserHaveError( user );
 
+const allUsersReady = ( users: GSuiteNewUser[] ): boolean =>
+	0 < users.length && users.every( userIsReady );
```

Comment:
Would it be possible to add a docblock comment to the `userIsReady()` and `allUsersReady()` functions to clarify what _ready_ means? Apparently it's just a way to determine if the data entered for a given G Suite user is correct, right?

Reply:
I think `userIsReady` => `userIsValid` would help clarify, and I can clarify via the comment as well.